--- a/client/mysql_secure_installation.cc	2025-03-31 04:00:22.000000000 -0400
+++ b/client/mysql_secure_installation.cc	2025-04-23 10:03:17.233170135 -0400
@@ -814,8 +814,11 @@
   }
 
   if (!hadpass) {
-    fprintf(stdout, "Please set the password for %s here.\n", opt_user);
-    set_opt_user_password(component_set);
+    fprintf(stdout,
+            "\nSkipping password set for %s as authentication with auth_socket is used by default.\n"
+            "If you would like to use password authentication instead, this can be done with the \"ALTER_USER\" command.\n"
+            "See https://dev.mysql.com/doc/refman/8.0/en/alter-user.html#alter-user-password-management for more information.\n\n"
+            , opt_user);
   } else if (!opt_use_default) {
     char prompt[256];
     fprintf(stdout, "Using existing password for %s.\n", opt_user);
